docs/migrate.md

```diff
@@ -4,6 +4,10 @@ This document records manual migrations that are necessary to apply when upgradi
 Sourcegraph versions. All manual migrations between the version you are upgrading from and the
 version you are upgrading to should be applied (unless otherwise noted).
 
+## 3.8
+
+If you're deploying Sourcegraph into a non-default namespace, refer to  [docs/configure.md](configure.md) for further configuration instructions.
```

Comment:
```suggestion
If you're deploying Sourcegraph into a non-default namespace, refer to ["Use non-default namespace in docs/configure.md](configure.md#use-non-default-namespace) for further configuration instructions.
```

You'll need to check to see if the hyperlink still works.

Reply:
done

Reply:
@uwedeportivo did you apply this diff?